tcms/xmlrpc/tests/test_version.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# pylint: disable=attribute-defined-outside-init, invalid-name
+# pylint: disable=attribute-defined-outside-init, invalid-name, avoid-list-comprehension
```

Comment:
FYI when there is only 1 thing you want to ignore, place the disable right besides it.